### DISCUSSION TITLE
Issue with ./gradlew generateTypes

### DISCUSSION THREAD

**Discussion by anuth212** [TEAMMATES/teammates#12398] [Help requests]
Hi, I am trying to get the front-end dev server to work. However, when I run ./gradlew generateTypes I am faced with this error output.
![image](https://user-images.githubusercontent.com/99857290/233236134-ae4967f2-9056-41c3-b38f-5e568baebc76.png)

Running the command always aborts at 42% when generateApiConstants is run. I followed the setup correctly so I am not sure why this would be the case.

**Comment by zhaojj2209** [comment]
Can you run the command again with `--stacktrace` so that we can get a more detailed error message? From the screenshot it seems that the task is failing on `FeedbackQuestionDetails`, but the error type as well as line number is not shown.
